string test_name
string[] joint_names
float32[] dither_amps
HoldPositionData[] hold_data # Data for each hold
---
